content/en/docs/contribute/generate-ref-docs/kubernetes-api.md

```diff
@@ -186,21 +182,18 @@ git submodule update --init --recursive --depth 1 # if not already done
 make container-serve
 ```
 
-## Commit the changes
+## Committing the changes
```

Comment:
The imperative form (“Commit”) feels more idiomatic

Reply:
Well, reverted, thanks.